src/aura/CRLP_RollupsContainer/CRLP_RollupsContainerHelper.js

```diff
@@ -40,9 +40,42 @@
         }
     },
 
+    /**
+     *  @description: merges and saves the updated row item into the existing list of rows
+     *  @param list - list of items to merge into
+     *  @param item - item to be merged into the list
+     *  @param context - context running the merge
+     */
+    mergeRowItem: function(cmp, list, item, context){
+        var newItem = true;
+        for (var i = 0; i < list.length; i++) {
+            //todo: refactor to lowercase Id after CMT save information is added
```

Comment:
Why refactor the Id to lowercase?

Reply:
Ah, this comment needs to be removed actually. There was a weird issue with the key of Id being uppercase in only some cases, but recordId looks to have resolved the problem I was seeing.